res/skins/LateNight/effect_unit.xml

```diff
@@ -93,11 +93,12 @@
                   <Text>ON</Text>
                 </State>
                 <Connection>
-                  <ConfigKey persist="true"><Variable name="group"/>,enabled</ConfigKey>
```

Comment:
O.o I don't understand either. This whole `<Connection>` element should probably be removed.